Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -98,7 +102,18 @@ public void setTargetSpeedForRange(double rangeInInches) {
     }
     
     public boolean isWheelAtSpeed() {
-        return super.getSpeed() >= super.getTargetSpeed() * wheelSpeedThresholdPercentage.get();
+        // Speed is in native units and can be negative
+        double currentSpeed = Math.abs(super.getSpeed());
+        if (currentSpeed <= 1) {
```

Comment:
But why is it needed? If our target speed is zero, then having a current speed of zero _is_ on-target. That state may indicate that the user chose an incorrect target, but I don't see a reason to special-case it as not being at the target speed when it clearly is for the domain of this system.